src/users/list/v2.rs

```diff
@@ -0,0 +1,108 @@
+//! [GET /_synapse/admin/v2/users/:user_id](https://github.com/matrix-org/synapse/blob/master/docs/admin_api/user_admin_api.rst#list-accountshttps://github.com/matrix-org/synapse/blob/master/docs/admin_api/user_admin_api.rst#query-user-account)
+
+use crate::serde::boolean_as_uint;
+use js_int::UInt;
+use ruma::api::ruma_api;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct UserMinorDetails {
```

Comment:
Can you move this struct to the end of the file? Usually the enpdoint definition is the first thing after the imports.